packages/material-ui/src/Button/Button.d.ts

```diff
@@ -7,6 +7,12 @@ import { OverrideProps, OverridableComponent, OverridableTypeMap } from '../Over
 export interface ButtonPropsVariantOverrides {}
 export type ButtonVariantDefaults = Record<'text' | 'outlined' | 'contained', true>;
 
+export interface ButtonPropsColorOverrides {}
+export type ButtonColorDefaults = Record<'inherit' | 'primary' | 'secondary', true>;
+
+export interface ButtonPropsSizeOverrides {}
+export type ButtonSizeDefaults = Record<'small' | 'medium' | 'large', true>;
```

Comment:
I'm not sure there is value in exporting this type, it might be the opposite.

Reply:
I've inlined the values where they are used 👍 It also makes the code more readable